core/data/src/main/java/uk/gov/gchq/gaffer/data/element/Edge.java

```diff
@@ -183,36 +220,61 @@ public String toString() {
                 .build();
     }
 
+    @JsonPOJOBuilder(withPrefix = "")
     public static class Builder {
-        private final Edge edge = new Edge();
+        private Object source;
+        private Object destination;
+        private boolean directed;
+        private boolean reversed;
+        private String group = "UNKNOWN";
+        private final Properties properties = new Properties();
 
         public Builder group(final String group) {
-            edge.setGroup(group);
+            this.group = group;
             return this;
         }
 
+        @JsonTypeInfo(use = JsonTypeInfo.Id.CLASS, include = JsonTypeInfo.As.PROPERTY, property = "class")
         public Builder source(final Object source) {
-            edge.setSource(source);
+            this.source = source;
             return this;
         }
 
-        public Builder dest(final Object dest) {
-            edge.setDestination(dest);
+        @JsonTypeInfo(use = JsonTypeInfo.Id.CLASS, include = JsonTypeInfo.As.PROPERTY, property = "class")
+        public Builder destination(final Object destination) {
```

Comment:
This field is referred to as "destination" everywhere else in the code base. There will some migration required, so is it worth changing this so that this is consistent everywhere?